### PR TITLE
Release candidate 3.2.1

### PR DESCRIPTION
**Pull request by ndellingwood**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,27 @@
 # Change Log
 
+## [3.2.01](https://github.com/kokkos/kokkos/tree/3.2.01) (2020-11-17)
+[Full Changelog](https://github.com/kokkos/kokkos/compare/3.2.00...3.2.01)
+
+**Fixed bugs:**
+- Disallow KOKKOS_ENABLE_CUDA_RELOCATABLE_DEVICE_CODE in shared library builds [\#3332](ihttps://github.com/kokkos/kokkos/pull/3332)
```
Comment (dalg24) [inline CHANGELOG.md:7]:
```suggestion
- Disallow KOKKOS_ENABLE_CUDA_RELOCATABLE_DEVICE_CODE in shared library builds [\#3332](https://github.com/kokkos/kokkos/pull/3332)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,27 @@
...
+- Do not install libprinter-tool when testing is enabled [\#3313](https://github.com/kokkos/kokkos/pull/3313)
+- Fix restrict/alignment following refactor [\#3373](https://github.com/kokkos/kokkos/pull/3373)
+- Fix zero-length reductions [#\3364](https://github.com/kokkos/kokkos/pull/3364)
+  - Pthread zero-length reduction fix [\#3452](https://github.com/kokkos/kokkos/pull/3452)
+  - HPX zero-length reduction fix [\#3470](https://github.com/kokkos/kokkos/pull/3470)
+  - cuda/9.2 zero-length reduction fix [\#3580](https://github.com/kokkos/kokkos/pull/3580)
+- Fix multi-stream scratch [#\3269](https://github.com/kokkos/kokkos/pull/3269)
+- Intel fix: workaround compiler issue with using statement [\#3383](https://github.com/kokkos/kokkos/pull/3383)
```
Comment (dalg24) [inline CHANGELOG.md:15]: Maybe bring this one (\#3383) right after \#3373 since it is really a fixup for it

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,27 @@
...
+- Guard KOKKOS_ALL_COMPILE_OPTIONS if Cuda is not enabled [\#3387](https://github.com/kokkos/kokkos/pull/3387)
+- Do not include link flags for fortran linkage [\#3384](https://github.com/kokkos/kokkos/pull/3384)
```
Comment (dalg24) [inline CHANGELOG.md:17]:
```suggestion
- Do not include link flags for Fortran linkage [\#3384](https://github.com/kokkos/kokkos/pull/3384)
```

**Review by dalg24 — APPROVED** [review]
Make sure you at least fix the URL for \#3332 in the changelog before you merge

**Comment by ndellingwood** [comment]
@dalg24 all comments should be addressed and git history cleaned up

**Comment by ndellingwood** [comment]
Cross-referencing:

kokkos/kokkos-kernels#851 (KokkosKernels 3.2.1 release)
trilinos/Trilinos#8359 (Trilinos with 3.2.1 snapshots)

**Comment by dalg24** [comment]
> @dalg24 all comments should be addressed and git history cleaned up

I have already approved

**Pull request merged**